readthedocs/projects/views/private.py

```diff
@@ -496,12 +503,13 @@ def get_context_data(self, **kwargs):
         return context
 
 
-class ProjectUsersCreate(SuccessMessageMixin, ProjectUsersMixin, CreateView):
-    success_message = _("Invitation sent")
+class ProjectUsersCreate(ProjectUsersMixin, CreateView):
+    success_message = _("User added")
```

Comment:
I think the message "Invitation sent" is the correct here; since we are not _adding_ the user immediately, since the user has to accept the invitation first.